client/index.html

```diff
@@ -94,10 +94,10 @@ <h2 class="toggle-menu">
                     </li>
                 </ul>
                 <ul class="global-menu menu-secondary-wrapper">
-                    <li class="global-menu-item global-menu-item--pipe">
-                        <span class="label">Language:</span>
+                    <li class="global-menu-item">
                         <language-switch class="language-dropown-wrapper"></language-switch>
                     </li>
+                    <li class="global-menu-item--pipe">&nbsp;</li>
```

Comment:
When the 'global-menu-item--pipe' class is applied to the li element, an empty element shouldn't be necessary?

Reply:
1. It is unfortunately required otherwise the separator isn't displayed. I changed it a bit so that it's done CSS-side.
2. I prefered to use a single element for the divider. It deserves a separated element in my opinion. I just changed the class name to something more self-explaining "global-menu-divider".